types/types.go

```diff
@@ -29,3 +30,9 @@ type SpecialAddresses struct {
 	FeeSink     sdk.Address
 	RewardsPool sdk.Address
 }
+
+// LegercoreValidatedBlock for serialization
+type LegercoreValidatedBlock struct {
+	Blk   bookkeeping.Block
+	Delta ledgercore.StateDelta
+}
```

Comment:
So we'll run these vb files through a "convert" function to update tests when we change the interfaces from go-algorand types?

Reply:
I was thinking that vb could be deserialized to[ types.ValidatedBlock](https://github.com/algorand/indexer/blob/develop/types/types.go#L13)